src/clarity-angular/datagrid/_datagrid.clarity.scss

```diff
@@ -170,9 +175,37 @@
                     height: calc(100% + #{$clr-baselineRem_0_5 - 1});
                     width: 1px;
                     top: -$clr-baselineRem_0_25 + 1;
-                    left: $clr-baselineRem_0_5;
+                    left: 0;
                     background-color: $clr-lighter-midtone-gray;
                 }
+
+                .separator-dragger {
+                    display: block;
+                    position: absolute;
+                    width: $clr-baselineRem_0_5+1;
+                    right: -$clr-baselineRem_0_25;
+                    top: -$clr-baselineRem_0_25;
+                    background-color: transparent;
+                    border: none;
+                    cursor: col-resize;
+                    margin: 0;
+                    outline: none;
+                    padding: 0;
+                    height: calc(100% + #{$clr-baselineRem_0_5 - 1});
+                    z-index: map-get($clr-layers, dropdown-menu);
+                }
+                .separator-tracker {
```

Comment:
I am thinking about making it `datagrid-column-handle-tracker` instead of `datagrid-column-tracker` ... 🤔